src/main/java/org/killbill/billing/client/RequestOptions.java

```diff
@@ -25,6 +25,7 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableMultimap;
 import com.google.common.collect.Multimap;
+import java.util.Objects;
```

Comment:
@sbrossie I guess it's time we finally start using _modern_ Java APIs (we don't support Java 6 anymore anyways since 0.20.x)? 👵  😏  

Somewhat related to https://github.com/killbill/killbill/issues/1203.